src/components/Common/CaseStudyCards/CaseStudyBlocks.js

```diff
@@ -19,14 +19,23 @@ const StyledColumnImage = styled(Col)`
 const renderImage = data => data && <Image image={data} />
 
 /* The reason for this is to solve inconsistent CSS Styling Between gatsby develop and build.
-This solution ensures that strong is always bold for any environment */
+This solution ensures that strong is always bold and white for any environment */
 const StyledBodyPrimary = styled(BodyPrimary)`
+  color: ${({ theme, bpColorReverse }) =>
+    theme.colors[bpColorReverse ? 'opacityWhite' : 'white']};
+
```

Comment:
Using `opacity: ` didn't work here as it applied it to a whole paragraph, including `strong`. Adding an opacity in strong would applied double opacities. I had to use `color` to solve. I added the equivalent of `white opacity 0.5` in theme.